zedd-app/src/holidays.ts

```diff
@@ -0,0 +1,418 @@
+export async function getHolidays(
+  year: number,
+  countryCode: string,
+  stateCode?: string,
+): Promise<Date[]> {
+  if (countryCode !== '') {
+    let url = 'https://date.nager.at/api/v3/publicholidays/' + year + '/' + countryCode
+    const response = await fetch(url)
+    const holidays = await response.json()
```

Comment:
`const holidays:{counties: string[]; global: boolean,date: string}[] = await response.json()` to avoid casts later